### PR TITLE
Homework-1 

### PR DESCRIPTION
**Pull request by georgiStoychev17**
Georgi Stoychev 7MI0600157

### CODE REVIEW

File: SI/Sem.01/Zadachi1/Task11.cpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	//Zadacha 11 
+
+	int a, b;
+	cout << "Number 1: ";
+	cin >> a;
+	cout << "Number 2: ";
+	cin >> b;
+
+	cout << "Maximum: " << max(a, b) << endl;
+	cout << "Minimum: " << min(a, b) << endl;
```
Comment (GeorgiTerziev02) [inline SI/Sem.01/Zadachi1/Task11.cpp:16]: The point here is not to use math.h

File: SI/Sem.01/Zadachi1/Task7.cpp [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include <math.h>
+using namespace std;
+
+int main() {
+
+//Zadacha 7
+
+	int apples;
```
Comment (GeorgiTerziev02) [inline SI/Sem.01/Zadachi1/Task7.cpp:9]: badly formated code - there should be indent

File: SI/Sem.01/Zadachi1/Task8.cpp [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <math.h>
+using namespace std;
+
+int main() {
+
+//Zadacha 8
+
+int radius;
```
Comment (GeorgiTerziev02) [inline SI/Sem.01/Zadachi1/Task8.cpp:9]: badly formated code

File: SI/Sem.01/Zadachi1/Task4.cpp [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <math.h>
+using namespace std;
+
+int main() {
+
+// Zadacha 4
+
+cout << "Please enter the length of the first side:" << endl;
```
Comment (GeorgiTerziev02) [inline SI/Sem.01/Zadachi1/Task4.cpp:9]: Badly formated code - there should be indent

**Review by GeorgiTerziev02 — COMMENTED** [review]
Looks good. I have left a few comments.

**Comment by GeorgiTerziev02** [comment]
We have already uploaded the solutions for week 1

**Comment by Justsvetoslavov** [comment]
Example solutions can be found [here](https://github.com/Justsvetoslavov/Introduction_to_programming/tree/master/SI/Sem.01/Pract.01)

**Pull request closed without merging**